README.md

```diff
@@ -1,3 +1,5 @@
+_[This repository has been moved to the main CLI repository][cli repo]_
```

Comment:
Link to repo? https://github.com/CircleCI-Public/circleci-cli

Reply:
It does link via the reference :)